server/src/pages/shot/text-tool.js

```diff
@@ -231,6 +232,17 @@ exports.TextTool = class TextTool extends React.Component {
     }
   }
 
+  onPaste(e) {
+    window.setTimeout(() => {
```

Comment:
Why a `setTimeout`?

Reply:
that's the recommendation to delay execution of script tp ensure paste event has completed updating the textContent